v4/source/brand.yaml

```diff
@@ -78,3 +78,30 @@
             }
 
             ok, resp := Client.UploadBrandImage(data.Bytes())
+
+    delete:
+      tags:
+        - brand
+      summary: Delete current brand image
+      description: |
+        Deletes the previously uploaded brand image. Returns 404 if no brand image has been uploaded.
+        ##### Permissions
+        Must have `manage_system` permission.
+      responses:
+        '200':
+          description: Brand image succesfully deleted
+          schema:
+            $ref: '#/definitions/StatusOK'
+        '404':
+          description: No brand image uploaded
+          $ref: '#/responses/NotFound'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
+
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            // GetBrandImage
+            img, err := Client.DeleteBrandImage()
```

Comment:
This doesn't return the image, as the API endpoint only returns an "OK" response if it succeeds, not the image.